### PR TITLE
Update faq.md - How to install YXI

### PR DESCRIPTION
**Pull request by ayx-david-ung**
How to install YXI

### CODE REVIEW

File: docs/references/faq.md [hunk 1]
```diff
@@ -295,3 +295,23 @@ To address this, use [-omit-ui in create-yxi](./ayx-plugin-cli#create-yxi).
...
+
+There are a few options to install YXIs.
+
+1. Right-click the Designer icon and select Run As Administrator. (Important Note: If you don’t Run as Administrator, the file won’t install to the correct folder)
+   1. In Designer, go to File > Open Workflow > Browse and select the downloaded YXI file.
+   2. Select Install for all users. (The files install to `C:\ProgramData\Alteryx\Tools`.)
+
+For Alteryx Server: Repeat the installation steps on all nodes or copy the folders to other nodes. You must install the Tool on all worker nodes. The Tool folder is all Server needs to use the connector so you can copy the folder between the nodes to install.
+  
+2. Double the the YXI.
```
Comment (aneeshnazar) [inline docs/references/faq.md]:
```suggestion
2. Double-click the the YXI.
```

File: docs/references/faq.md [hunk 2]
```diff
@@ -295,3 +295,23 @@ To address this, use [-omit-ui in create-yxi](./ayx-plugin-cli#create-yxi).
...
+   1. A pop-up will appear and you will be asked to confirm installation. Follow steps from pop-up.
+4. Drag the YXI into Designer Desktop Canvas
+   1. A pop-up will appear and you will be asked to confirm installation. Follow steps from pop-up.
+6. From the `ayx-plugin-cli` CLI, use the `designer-install` command.
```
Comment (aneeshnazar) [inline docs/references/faq.md]:
```suggestion
6. From the `ayx-plugin-cli` CLI, use the `install-yxi` command.
```
`designer-install` only works from a workspace - `install-yxi` works on arbitrary YXIs too

File: docs/references/faq.md (hunk 1 above)
Comment (ayx-david-ung) [reply docs/references/faq.md]: Updated

File: docs/references/faq.md (hunk 2 above)
Comment (ayx-david-ung) [reply docs/references/faq.md]: Updated

**Review by aneeshnazar — APPROVED** [review]

File: docs/references/faq.md [hunk 3]
```diff
@@ -295,3 +295,23 @@ To address this, use [-omit-ui in create-yxi](./ayx-plugin-cli#create-yxi).
 ## _"I don't want to rebuild UI components when installing a YXI through the CLI."_
 
 To address this, use [-omit-ui in designer-install](./ayx-plugin-cli#designer-install).
+
+## _"How do you install YXI packages into Alteryx Designer Desktop?"_
+
+There are a few options to install YXIs.
+
+1. Right-click the Designer icon and select Run As Administrator. (Important Note: If you don’t Run as Administrator, the file won’t install to the correct folder)
+   1. In Designer, go to File > Open Workflow > Browse and select the downloaded YXI file.
```
Comment (paul-england) [inline docs/references/faq.md:304]: Is this correct?  You select? I would assume if you select Open Workflow you're limited to opening workflow files.

File: docs/references/faq.md [hunk 4]
```diff
@@ -295,3 +295,23 @@ To address this, use [-omit-ui in create-yxi](./ayx-plugin-cli#create-yxi).
...
+   2. Select Install for all users. (The files install to `C:\ProgramData\Alteryx\Tools`.)
+
+For Alteryx Server: Repeat the installation steps on all nodes or copy the folders to other nodes. You must install the Tool on all worker nodes. The Tool folder is all Server needs to use the connector so you can copy the folder between the nodes to install.
```
Comment (paul-england) [inline docs/references/faq.md]: I don't think "Tool" should be capitalized.

File: docs/references/faq.md (hunk 3 above)
Comment (ayx-david-ung) [reply docs/references/faq.md:304]: Correcting. Got feedback from Designer Team

File: docs/references/faq.md (hunk 4 above)
Comment (ayx-david-ung) [reply docs/references/faq.md]: Corrected

**Review by paul-england — APPROVED** [review]

**Review by ayx-orion — APPROVED** [review]
lgtm

**Pull request merged**